docs/containers/debug-netcore.md

```diff
@@ -19,16 +19,34 @@ MetaDescription: Debug a .NET app running in a Docker container, using Visual St
 
 ## Walkthrough
 
-1. If needed, create a .NET project with `dotnet new`.
-1. Open the project folder in VS Code.
+- If needed, create a .NET project with `dotnet new`.
+- Open the project folder in VS Code.
+- Optionally, set a breakpoint.
+
+## .NET SDK vs. Dockerfile Build
+
+There are two ways to build and debug your app inside a container: using a Dockerfile or, for .NET 7 and above, without a Dockerfile.
```

Comment:
```suggestion
There are two ways to build and debug your app inside a container: using a Dockerfile or, for .NET 7 and later, without a Dockerfile.
```

Reply:
Normal practice is to use time language (earlier, later) rather than physical location language (above, below).